nvflare/tool/job/config/configer.py

```diff
@@ -288,10 +299,12 @@ def build_config_file_indices(job_folder: str, app_names: List[str]) -> Dict[str
     return app_config_file_index
 
 
-def get_app_name_from_path(path):
+def get_app_name_from_path(path: str):
     # path is in the format of as app1/xxx.conf
     # path app1/xxx.conf
     # xxx.conf
+    if _is_meta_file(path):
```

Comment:
Here the path could include the directory, while `_is_meta_file()` currently assumes just a filename

Reply:
good catch, let me fix this